src/libMediator/Mediator.cpp

```diff
@@ -242,3 +243,17 @@ void Mediator::IncreaseEpochNum() {
 }
 
 bool Mediator::GetIsVacuousEpoch() { return m_isVacuousEpoch; }
+
+uint32_t Mediator::GetShardSize(const bool& useShardStr) const {
```

Comment:
useShardStr may not be intuitive and may be confusing

Reply:
OK, I will change to a better name